lcoe_calculator/base_processor.py

```diff
@@ -308,12 +316,13 @@ def _append_reference_info(
         if len(df_ref) == 0:
             raise ValueError(
                 f"There is no reference for year {year} for metric '{metric}', scenario "
-                f"'{scenario}'"
+                f"'{scenario}, and tech detail '{tech_detail}'"
             )
         if len(df_ref) > 1:
```

Comment:
This error feels a little overzealous at the moment. Is it possible to filter such that this only appears when a single combination of tech detail, year, and scenario have multiple references? Would a test file be helpful?